tests/Fixer/PhpUnit/PhpUnitAttributesFixerTest.php

```diff
@@ -0,0 +1,560 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\PhpUnit;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\PhpUnit\PhpUnitAttributesFixer
+ */
+final class PhpUnitAttributesFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @requires PHP 8.0
+     *
+     * @dataProvider provideFixCases
+     */
+    public function testFix(string $expected, ?string $input = null): void
+    {
+        $this->doTest($expected, $input);
+    }
+
+    /**
+     * @return iterable<array{0: string, 1?: string}>
+     */
+    public static function provideFixCases(): iterable
+    {
+        yield 'do not fix with wrong values' => [<<<'PHP'
+            <?php
+            /**
+             * @requires
+             * @uses
+             */
+            class FooTest extends \PHPUnit\Framework\TestCas {
+                /**
+                 * @backupGlobals
+                 * @backupStaticAttributes
+                 * @covers
+                 * @dataProvider
+                 * @depends
+                 * @group
+                 * @preserveGlobalState
+                 * @testDox
+                 * @testWith
+                 * @ticket
+                 */
+                public function testFoo() { self::assertTrue(true); }
+            }
+            PHP];
+
+        yield 'do not fix with wrong casing' => [<<<'PHP'
+            <?php
+            /**
+             * @COVERS \Foo
+             */
+            class FooTest extends \PHPUnit\Framework\TestCas {
+                /**
+                 * @dataPROVIDER provideFooCases
+                 * @requires php 8.3
+                 */
+                public function testFoo() { self::assertTrue(true); }
+            }
+            PHP];
+
+        yield 'do not fix when not supported by attributes' => [<<<'PHP'
+            <?php
+            /**
+             * @covers FooClass::FooMethod
+             * @uses ClassName::methodName
+             */
+            class FooTest extends \PHPUnit\Framework\TestCas {
```

Comment:
Are the typos (`TestCas`) here intentional?

Reply:
Those are typos 😄